CHANGES.rst

```diff
@@ -222,6 +257,10 @@ tweakreg
 
 - Refactored code to work with changes in ``tweakwcs`` version 0.8.0. [#7006]
 
+source_catalog
```

Comment:
Some of these change log entries were originally entered long enough ago that they show up under the wrong release headings, because they didn't actually get merged until after 1.7.2 was released and hence now belong up in the top section for 1.7.3 development. I think that's been fixed in the version that's on master now. We can always fix any conflicts separately from this PR.